doc/content/reference/id-eui-constraints/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "ID and EUI Constraints"
+description: ""
+---
+
+This reference explains how {{% tts %}} deals with IDs and EUIs, including regular expression requirements, uniqueness, and how deletion is handled.
+
+<!--more-->
+
+## Requirements of an ID or EUI
+
+An ID or EUI in {{% tts %}} must:
+
+- Have a length of between 3 and 36 characters (inclusive)
+- Consist of lowercase letters, numbers, and non-consecutive dashes
+- **NOT** begin or end with a dash
+
+The following regular expression is used to validate IDs and EUIs:
+
+`(^[a-z0-9](?:[-]?[a-z0-9]){2,}$)`
+
+## Uniqueness
+
+Within a deployment, there can only be:
+
+- One tenant with the same ID
+- One gateway with the same EUI (across all tenants)
+- One end device with the same DevEUI,JoinEUI tuple (across all tenants)
```

Comment:
Maybe "DevEUI,JoinEUI tuple" is too technical, you can also write "combination of JoinEUI and DevEUI"